webapp/mapTest4.html

```diff
@@ -100,16 +100,16 @@
             <li><a id="button_press6" onclick="pressBtn6()">Cafeteria</a></li>
             <li><a id="button_press7" onclick="pressBtn7()">The Carl and Celia Berta Gellert Library</a></li>
             <li><a id="button_press8" onclick="pressBtn8()">Library Lawn</a></li>
-            <li><a id="button_press9" onclick="pressBtn9()">Walter Gleason Gym </a></li>
-            <li><a id="button_press10" onclick="pressBtn10()">Campus Center</a></li>
-            <li><a href="/">Academic Success Center</a></li>
-            <li><a href="/">International Student Office</a></li>
-            <li><a href="/">Student Life & Leadership Office</a></li>
-            <li><a href="/">The Quad</a></li>
-            <li><a href="/">Writing Center</a></li>
-            <li><a href="/">Cunningham Chapel Annex</a></li>
-            <li><a href="/">Center for Spirituality</a></li>
-            <li><a href="/">Office of Diversity</a></li>
+            <li><a id="button_press9" onclick="pressBtn9()">Walter Gleason Gym</a></li>
```

Comment:
That's very odd.  There is nothing saying you can't name a function or id an element with 10.  But see my other comments about descriptive naming practices.  THat may be the better solution.